app/src/main/res/layout/onboarding_slide_final.xml

```diff
@@ -46,8 +46,7 @@
         android:gravity="center_horizontal"
```

Comment:
Apply all the changes to this file which I have mentioned previously.

Reply:
Made the change!